packages/blocks/src/ColumnLayout/ColumnLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Flex, Link, TextPairing } from '@cmpsr/components';
+import { ColumnLayoutType } from './types';
+import { MediaBlock } from '../';
+
+export const ColumnLayout: ColumnLayoutType = (props) => <Flex flexDirection="column" {...props} />;
+
+ColumnLayout.ColumnGroup = Flex;
+ColumnLayout.Column = MediaBlock;
```

Comment:
we should the variant to vertical by default, in here or in the `MediaBlock` component

Reply:
Updated `MediaBlock` to have `vertical` as default value of `variant`